servicetalk-concurrent-api/docs/modules/ROOT/pages/async-context.adoc

```diff
@@ -150,8 +150,8 @@ publisher.flatMapMergeSingle(v -> {
   Integer contextValue = AsyncContext.get(KEY);
   assert contextValue == 10 || contextValue == 30; // (2) Subscriber chain may see either value.
 
-  // AsyncContext will be copied/isolated when this async source is subscribed to. Changes to the AsyncContext map from
-  // async operators on inner operator chain will therefore not be visible in the outer operator chain.
+  // AsyncContext will be copied when Single.subscribe(..) is called. Changes to the AsyncContext map from operators on
+  // the inner Single operator chain will therefore not be visible in the outer Publisher operator chain.
```

Comment:
You may wish to add as a comment where the `shareContextOnSubscribe()` would go if they wanted a shared context.

Reply:
Lets consider this as a followup PR. I don't want to overload concepts in this PR.